components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,374 @@
+/* eslint-disable camelcase */
+import axios from "axios";
+import get from "lodash/get.js";
+import sortBy from "lodash/sortBy.js";
+import isArray from "lodash/isArray.js";
+import isEmpty from "lodash/isEmpty.js";
+import isString from "lodash/isString.js";
+import zoomCountries from "./zoom_countries.mjs";
+
+export default {
+  type: "app",
+  app: "zoom_admin",
+  propDefinitions: {
+    cloudRecording: {
+      type: "string",
+      label: "Recording",
+      description: "The ID of the cloud recording. Please use a valid object with `meetingId` and `value` with \"Structured Mode\" disabled. `value` means the recording ID",
+      async options({
+        page,
+        prevContext,
+      }) {
+        const data = await this.listUserRecordings(
+          prevContext.nextPageToken,
+          page + 1,
+        );
+
+        if (!data.meetings) {
+          return [];
+        }
+        const options = [];
+        data.meetings.forEach((meeting) => {
+          meeting.recording_files.forEach((recording) => {
+            const label = `${meeting.topic} - (Format: ${recording.file_type}) (From ${recording.recording_start} to ${recording.recording_end})`;
+            options.push({
+              label,
+              value: {
+                meetingId: meeting.id,
+                label,
```

Comment:
I think is not necessary to set the label inside `value` right?

Reply:
Please check my comment in `zoom_admin.app.mjs` :)